environment.yml

```diff
@@ -9,7 +9,7 @@ channels:
 - tethysplatform
 - tethysplatform/label/dev
 - conda-forge
-- bokeh/label/dev
+- bokeh
```

Comment:
@msouff Do we need the bokeh channel? I believe our goal is to eventually move to conda-forge exclusively.

Reply:
2.0.0 is available from conda-forge now. I think when I fixed this bokeh 2.0.0 was coming from the bokeh dev channel and I noticed it was also in the main bokeh channel and that's probably why I got rid of the bokeh dev channel. But now that it is in conda-forge too, we don't need that one either.